workshop-example-aws.yaml

```diff
@@ -15,9 +15,11 @@ workshop:
 
     # The workshop cloud provider & VM configuration
     cloud_provider: aws
-    access_key: <access key>
-    secret_key: <secret key>
-    region: eu-west-2
+    #access_key: <access key>
+    #secret_key: <secret key>
+    profile: <AWS profile>
+    region: <AWS preferred region>
+    availability_zones: <list of availability zones>
```

Comment:
so here you have the availability zones, so you could use those in the VPC I guess? @moliele

Reply:
@korzan-og Did you mean in this comment: https://github.com/confluentinc/confluent-hybrid-cloud-workshop/pull/63#discussion_r1072097897 , adding these lines? thanks!

Reply:
Yes. Thank you.